time/src/main/java/io/spine/time/ZoneOffsets.java

```diff
@@ -205,7 +205,8 @@ void check(int value) {
              */
             @Override
             void checkReduced(int value) {
-                DtPreconditions.checkBounds(value, name().toLowerCase(), min() + 1, max() - 1);
+                DtPreconditions.checkBounds(value, name().toLowerCase(Locale.ENGLISH), min() + 1,
```

Comment:
`name().toLowerCase(Locale.ENGLISH)`

I would extract it to a variable.

Reply:
I've extracted a method.